include/client/networkclient.h

```diff
@@ -61,7 +61,8 @@ class NetworkClient : public QObject {
       QString client_version,
       QString client_description,
       QString client_type,
-      QByteArray authentication_key);
+      QByteArray authentication_key,
```

Comment:
Should be `const QByteArray&`. While you're editing this method, please also fix the rest of arguments (same problem).